common/check_free_space

```diff
@@ -9,28 +9,29 @@
 # - Vincent Garonne, <vincent.garonne@cern.ch>, 2015
 
 '''
-Probe to check the min free space.
+Probe to check the free space.
```

Comment:
This is the one which I worried was stepping on ATLAS toes. What I'd suggest is to just make your version and rename it as check_report_free_space or something like that. Then we can run your version.